README.md

```diff
@@ -37,6 +37,10 @@ Ensure docker is running. Spin up MySQL locally:
 ```
 $ ./docker/run-mysql.sh start leonardo  
 ```
+If you have an M1 laptop, pull the x86_64 mysql image:
+```
+docker pull --platform linux/x86_64 mysql:5.6
```

Comment:
running m1, we need to force pull the linux/x86_64 image because the linux/arm64 one does not exist for 5.6
![Screen Shot 2022-11-17 at 12 23 27 PM](https://user-images.githubusercontent.com/5375000/202514917-c24d28a2-2416-4f5b-a253-86129fe4f86a.png)

I'll make that more explicit!

Reply:
I guess the other way to solve this is to add the docker pull command to https://github.com/DataBiosphere/leonardo/blob/develop/docker/run-mysql.sh

Reply:
pulling this into a new PR https://github.com/DataBiosphere/leonardo/pull/3023